plugin_config.yml

```diff
@@ -7,4 +7,4 @@ pluginType: STAGED
 entryPoint: plugin_runner:plugin
 srcDir: src
 schemaFile: schema.json
-buildNumber: 2.0.26
+buildNumber: 2.0.40
```

Comment:
I think we agreed on using 2.1.0,right?

Reply:
I thought we can make that so in the main branch when we do the final merge. Thoughts?